src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -526,20 +521,12 @@ function(Comment $comment) {
         );
     }
 
-    /**
-     * Get a count per comment
-     *
-     * @return array
-     */
     public static function getCommentStatusCount(): array
     {
-        return (array) BackendModel::getContainer()->get('database')->getPairs(
-            'SELECT i.status, COUNT(i.id)
-             FROM blog_comments AS i
-             WHERE i.language = ?
-             GROUP BY i.status',
-            [BL::getWorkingLanguage()]
-        );
+        $repository = BackendModel::get('doctrine.orm.default_entity_manager')
+                                  ->getRepository(Comment::class);
```

Comment:
I would create a service for this, you're gonna want to inject the repository in a later phase anyway

Reply:
Fixed in cd9cef81ef540cf410bc31750966a00591aa06ae